docker/Dockerfile.benchmark-container-prometheus-export

```diff
@@ -1,4 +1,8 @@
-FROM alpine as builder
+# Leave empty for x86, set to e.g. "arm64v8/" (note closing slash!) for ARM, e,g, 
+# $ docker build --build-arg ARCH="arm64v8/" -t wrk2-arm -f docker/Dockerfile.benchmark-container-prometheus-export .
+ARG ARCH=""
+
+FROM ${ARCH}alpine as builder
```

Comment:
Shouldn't the same thing be done for the other alpine image?

Reply:
It should! Nice catch, worked on my machine b/c I use qemu via binfmt-misc, so the ARM wrk2 ran on the x86 Alpine w/o issues. Of course, not having ARCH in the target image would break things on "real" ARM machines. Will fix.

Reply:
Done, added `ARCH` to the production container image too.